### PR TITLE
dsv: Use correct dict usage

### PR DESCRIPTION
**Pull request by Akasurde**
##### SUMMARY

Signed-off-by: Abhijeet Kasurde <akasurde@redhat.com>

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
changelogs/fragments/dsv_fix.yml
plugins/lookup/dsv.py

### CODE REVIEW

**Comment by felixfontein** [comment]
CC @amigus 

@amigus is there a reason your github nick is not included in the `author` field? This way, you will never be notified of issues and PRs for your plugins.

File: changelogs/fragments/dsv_fix.yml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+- dsv - use correct dict usage.
```
Comment (felixfontein) [inline changelogs/fragments/dsv_fix.yml:2]:
```suggestion
- dsv lookup - use correct dict usage (https://github.com/ansible-collections/community.general/pull/743).
```

**Comment by amigus** [comment]
> CC @amigus
> 
> @amigus is there a reason your github nick is not included in the `author` field? This way, you will never be notified of issues and PRs for your plugins.

No, there's no reason. I must have done something wrong along the way to remove it.  How can I add it (back)?

**Comment by felixfontein** [comment]
@amigus good to know! I created #753 to fix that.

**Comment by amigus** [comment]
> @amigus good to know! I created #753 to fix that.

Cool, thanks -- much appreciated!  Please let me know if you need anything else from me.  I'm really looking forward to seeing these things ship with 2.10, a few people are already using them!

**Comment by felixfontein** [comment]
@amigus could you please check this PR whether the change is correct?

**Comment by amigus** [comment]
> @amigus could you please check this PR whether the change is correct?

Oh, right, sorry, yes, it is correct.

**Comment by felixfontein** [comment]
@Akasurde thanks for fixing this!
@amigus thanks for reviewing!

**Pull request merged**